### PR TITLE
Removing auto generated comment markup around includes

### PR DESCRIPTION
**Pull request by DuncanmaMSFT**
Prevents breaking output when includes used outside of body content
- removes comments from page that could end up in production HTML
- errors in include still produce comments as this could aid debugging

### CODE REVIEW

File: test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs [hunk 1]
```diff
@@ -796,9 +796,8 @@ public void TestBuildWithInvalidPathWithTokenAndMapping()
                         "\n",
                         "<p><a href=\"invalid-a.md\">link a</a>",
                         "<a href=\"../b/invalid-b.md\">link b</a></p>",
-                        $"<!-- BEGIN INCLUDE: Include content from &quot;{_inputFolder}/b/token.md&quot; --><p><a href=\"invalid-a.md\">link a</a>",
-                        "<a href=\"../b/invalid-b.md\">link b</a></p>",
-                        "<!--END INCLUDE -->"),
+                        $"<p><a href=\"invalid-a.md\">link a</a>",
+                        "<a href=\"../b/invalid-b.md\">link b</a></p>"),
```
Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs]: ><a href=\"../b/invalid-b.md\">link b</a></p>"), [](start = 25, length = 47)

ut failure: 
Expected: ï¿½ï¿½ï¿½b.md">link b</a></p>
Actual:   ï¿½ï¿½ï¿½b.md">link b</a></p>\n

Comment (DuncanmaMSFT) [inline test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs]: Ach, sorry... I fixed a ton of tests, but I can't actually run them locally for some reason, whenever I try I get an error 

CS1705	Assembly 'Microsoft.DocAsCode.Build.Engine' with identity 'Microsoft.DocAsCode.Build.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' uses 'jint, Version=2.10.3.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f' which has a higher version than referenced assembly 'Jint' with identity 'Jint, Version=2.5.0.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f'	Microsoft.DocAsCode.Build.Engine.Tests..NET Framework 4.5.2	C:\Repos\CSI\docfx\test\Microsoft.DocAsCode.Build.Engine.Tests\ConvertStrongTypeToJsValueTest.cs	22	Active

**Comment by DuncanmaMSFT** [comment]
@vicancy Thanks for your earlier review, I've updated the tests to match the new output (after your help getting the build working on my machine as well) and pushed up those new changes.

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**